### DISCUSSION TITLE
How to get the builder running?

### DISCUSSION THREAD

**Discussion by ile** [alibaba/formily#3769] [Ideas]
Hi there,

Firstly, Formily looks really good. Thanks for it.

Then, It's not clear how to get the Form Builder to run. The [docs](https://formilyjs.org/guide/form-builder) don't really help - the source code on that page will not run and it's not clear how to fix it.

Can someone help a bit? Maybe improve the docs too? Thanks.

I was able to get this running: https://github.com/pindjs/designable

**Comment by spt110** [comment]
1.npm install yarn -g
2.yarn install
3.yarn workspace @designable/formily-antd start

**Comment by ile** [comment]
Thanks. With that I get this (and a blank screen):

![image](https://user-images.githubusercontent.com/433707/227701678-4e526ac6-c76f-4955-a610-dddfbf46c7cd.png)

Actually I got this earlier too so I went to look for other ways to run it.

Is there a quick fix for this?

**Reply by spt110** [reply]
<script src="https://unpkg.com/antd/dist/antd-with-locales.min.js"></script>    
replace
  <script src="https://unpkg.com/antd@4.15.2/dist/antd-with-locales.min.js"></script>

**Comment by ile** [comment]
Thanks, now it works!
